build.sh

```diff
@@ -16,6 +16,7 @@ DOCKERFILE_PATH=""
 # Perform docker build but append the LABEL with GIT commit id at the end
 function docker_build_with_version {
   local dockerfile="$1"
+  [ -n "$(echo "${EXCLUDE_VERSIONS}" | grep ${dir}.${OS})" ] && echo "-> $OS build of version $dir found in EXCLUDED_VERSIONS, skipping build." && return
```

Comment:
I'm just thinking whether we can restrict the regexp a bit, since if we ever had something like such two files `Dockerfile.fedora-module` and `Dockerfile.fedora` in the repo, while having `Dockerfile.fedora-module` in `EXCLUDE_VERSIONS`, `echo "${EXCLUDE_VERSIONS}" | grep ${dir}.${OS}` would match both `Dockerfile.fedora` and `Dockerfile.fedora-module`..

`grep -e "^\(.*\s\)\?${dir}.${OS}\(\s.*\)\?$"` looks terrible though..